source/game/kart/KartMove.cc

```diff
@@ -638,6 +669,22 @@ void KartMove::resetDriftManual() {
     m_mtCharge = 0;
 }
 
+/// @stage 2
+/// @addr{0x80584B58}
+void KartMove::clearDrift() {
+    m_outsideDriftAngle = 0.0;
```

Comment:
```suggestion
    m_outsideDriftAngle = 0.0f;
```
A `0.0` literal is double-precision by default, which would then be casted down into a single-precision number to store it. This is likely optimized down in x86, but is still a good idea to specify.